automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -239,9 +239,26 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
           }
         }
       }
-
     }
 
+    "should install multiple user specified notebook extensions" in withWebDriver { implicit driver =>
+      val clusterName = ClusterName("user-jupyter-ext" + makeRandomId())
+      withCleanBillingProject(hermioneCreds) { projectName =>
+        val project = GoogleProject(projectName)
+        implicit val token = hermioneAuthToken
+        withNewCluster(project, clusterName, ClusterRequest(userJupyterExtensionConfig = Some(multiExtensionClusterRequest))) { cluster =>
+          withNewNotebook(cluster, Python3) { notebookPage =>
+            //Check if the mark up was translated correctly
+            val nbExt = notebookPage.executeCell("! jupyter nbextension list")
+            nbExt.get should include("jupyter-gmaps/extension  enabled")
+            nbExt.get should include("pizzabutton/index  enabled")
+            nbExt.get should include("my_ext/main  enabled")
+            val serverExt = notebookPage.executeCell("! jupyter serverextension list")
+            serverExt.get should include("pizzabutton  enabled")
+            serverExt.get should include("jupyterlab  enabled")
+          }
+        }
+      }
+    }
```

Comment:
It may be good to add unit tests for some corner case scenarios such as permutations of one or more unspecified extensions as well as error cases such as invalid extension specifications.